torchao/quantization/quant_api.py

```diff
@@ -223,11 +228,15 @@ def change_linear_weights_to_int4_woqtensors(model, **kwargs):
     """
     filter_fn = kwargs.pop("filter_fn", _is_linear)
 
-    _replace_with_custom_fn_if_matches_filter(
-        model,
-        _get_subclass_inserter(Int4WeightOnlyQuantizedLinearWeight, enable_parametrization=TORCH_VERSION_AFTER_2_4, **kwargs),
-        filter_fn,
-    )
+    if TORCH_VERSION_AFTER_2_4:
+        quantize(model, get_apply_int4wo_quant(**kwargs), filter_fn)
```

Comment:
blind kwargs make it impossible to document the behavior. i understand that `change_linear_weights_to_int4_woqtensors` has this as well. Seems like something that could be worth fixing.

Reply:
yeah sure